docs/guide/README.md

```diff
@@ -23,7 +22,7 @@ and add the first Content entries to the freshly created space.
 
 #### 4. Download app to display your content
 
-A shortcut to [contentful boilerplate](../boilerplate) to download a boilerplate
+A shortcut to download a [boilerplate example project](https://github.com/contentful?q=boilerplate&type=all&language=&sort=) to download a boilerplate
 example project built with one of our SDKs.
```

Comment:
Is this still correct? 
It mentions boilerplate, but links to Contentful GitHub (and there's some duplicate text in the new version)

Reply:
Good question. I changed the text now to be more meaningful. Would you leave it like it is in the new version or remove it altogether?

Reply:
Sounds great now 👍